styles/layout.css

```diff
@@ -0,0 +1,66 @@
+
+
+
+#menu {
+  margin: 0;
+  float: left;
+  display: inline-block;
+  width: 200px;
+}
+#main {
+  margin-left: 200px;
+  padding: 0px;
+}
+
+
+#contact-list {
+  padding-left: 30px;
+  display: none;
+}
+#contact:hover + #contact-list {
+  display: block;
+}
+
+#contact ul li{
+  display: none;
+}
+#contact:hover ul li{
+  display: block;
+  padding-left: 30px;
+}
+
+@media screen and (max-width: 400px) {
+  #menu {
+    margin: 0;
```

Comment:
since you have this same style above in line 5, you don't need to include it here too (it's not changing).